python/modules.py

```diff
@@ -0,0 +1,11 @@
+import libraries.FileIO
+import libraries.json
+import libraries.request
+import libraries.SystemIO
+import libraries.times
+import libraries.websocket
+import sys
+
+libraries = sys.modules
+
+del libraries["sys"]
```

Comment:
Hmm, why not just make this a dictionary, like

```py
libraries = {
  'FileIO': libraries.FileIO,
  # ...
}
```

Seems like it'd be a less hacky approach